ui/jobs/buff_tracker.ts

```diff
@@ -1,38 +1,36 @@
-import { JobsOptions } from './types';
-
-import WidgetList from '../../resources/widget_list';
 import EffectId from '../../resources/effect_id';
-import { MatchesAbility, MatchesGainsEffect, MatchesLosesEffect } from '../../resources/matches';
-import PartyTracker from '../../resources/party';
-
-import potionImage from '../../resources/ffxiv/status/potion.png';
-import astralImage from '../../resources/ffxiv/status/astral.png';
-import umbralImage from '../../resources/ffxiv/status/umbral.png';
-import physicalImage from '../../resources/ffxiv/status/physical.png';
-import offguardImage from '../../resources/ffxiv/status/offguard.png';
-import peculiarLightImage from '../../resources/ffxiv/status/peculiar-light.png';
-import trickAttackImage from '../../resources/ffxiv/status/trick-attack.png';
-import battleLitanyImage from '../../resources/ffxiv/status/battle-litany.png';
-import emboldenImage from '../../resources/ffxiv/status/embolden.png';
 import arrowImage from '../../resources/ffxiv/status/arrow.png';
+import astralImage from '../../resources/ffxiv/status/astral.png';
```

Comment:
Explicit callout for this file, if people care.

Reply:
What's wrong?

Reply:
Nothing is wrong, just pointing out this one since it has the img imports